pages/editor/new.tsx

```diff
@@ -16,6 +16,7 @@ const PublishArticleEditor = () => {
     body: "",
     tagList: [],
   };
+  const Title = React.createRef();
```

Comment:
We should let users add tags with another field. That logic already exists and can be found in the master branch since it has not been updated. I would recommend looking at the  `pages/editor/new.tsx` in the master branch to see how adding tags worked.

Reply:
added the tags component